docs/changes/wcs/15795.bugfix.rst

```diff
@@ -0,0 +1,7 @@
+Updated bundled WCSLIB version to 8.2.2. This update fixes character buffer
+overflows in the comment string for the longitude and latitude axes triggered
+by some projections in ``wcshdo()``, and also the formatting for generic
+coordinate systems. For a full list of changes - see
+http://www.atnf.csiro.au/people/mcalabre/WCS/CHANGES or
+`astropy/cextern/wcslib/CHANGES <https://github.com/astropy/astropy/blob/
```

Comment:
Maybe we don't have to link here? It is failing RTD build with a warning:

`docs/changelog.rst:5: WARNING: Duplicate explicit target name: "astropy/cextern/wcslib/changes".`

Reply:
I fixed this error but other warnings seem unrelated.